Example/android/build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         minSdkVersion = 21
         compileSdkVersion = 30
         targetSdkVersion = 30
-        ndkVersion = "20.1.5948944"
+        ndkVersion = "21.4.7075529"
```

Comment:
https://facebook.github.io/react-native/docs/building-from-source#download-links-for-android-ndk - for some reason they haven't updated the download link to the newer version

Reply:
yep, also saw it :) but I actually was surprised how smooth my upgrade to the new NDK was, not as painful as before :)